reactioncommerce/reaction
# refactor: import Reaction from /client|server|lib/api when possible

Simple refactor of `import { Reaction }` statements that were using a relative path instead of `/client/api` so that our Reaction imports are consistently styled.

**To Test**
1. Make sure the app starts
2. Layouts should load
3. Templates should load
4. `hasPermissions` works
